doc/en/deprecations.rst

```diff
@@ -12,13 +12,47 @@ should be used instead.
     :local:
 
 
+.. _deprecate-hook-marks:
```

Comment:
You probably intended this to be right above the `configuring hook specs/impls using markers` heading, not here.

Reply:
going to drop that one again